src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -18,9 +18,19 @@ class Lambda {
     databaseName match {
       case "consignmentapi" => createConsignmentApiUsers
       case "keycloak" => createKeycloakUser
+      case "bastion" => createBastionUser
     }
   }
 
+  def createBastionUser: Boolean = {
+    val user = createConsignmentApiUser(lambdaConfig.bastionUser)
+    //Grant access to tables created before we started using the migrations user
+    sql"GRANT SELECT ON ALL TABLES IN SCHEMA public TO $user;".execute().apply()
+    sql"SET ROLE migrations_user;".execute().apply()
```

Comment:
Should this role be different?

Reply:
No, migrations_user owns the tables because that's the one that flyway runs with. `SET ROLE` switches to that user which then allows you to grant access to tables it owns.